### DISCUSSION TITLE
Change default markdown preview font

### DISCUSSION THREAD

**Discussion by ftessier** [community/community#49735] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

The preview feature for markdown text edit boxes is extremely useful. It would be great though if the preview font was changed to "open sans", which has more distinct italics. I have trouble seeing the difference between roman and italic in the preview, which takes away that option for emphasis in my comments.

If there is a setting adjusting the preview font, I have not found it. It might be nice to have 2-3 options for the _preview_ font (for example: mono, sans, serif); but even just updating to a sans font that has distinctive italics would be fantastic!

**Comment by bbkx226** [comment]
At this time, there is no option to change the font used in the preview feature. However, your suggestion of using a font with more distinct italics, such as Open Sans, is a great idea that could improve the readability of the preview.
